packages/babel-plugin-undeclared-variables-check/src/index.js

```diff
@@ -1,17 +1,65 @@
 import leven from "leven";
 
+const envGlobals = {
```

Comment:
This PR seems reasonable to me, but rather than hard-code these, we'd be better off using a module like https://github.com/sindresorhus/globals/blob/master/globals.json#L824 which we also already use in `babel-traverse`.